src/lib/components/leaderboard/MeasureBigNumber.svelte

```diff
@@ -23,16 +24,22 @@
   };
   let bigNumber;
   $: bigNumber = $bigNumberEntity?.bigNumbers?.[$measureField] ?? 0;
-  const bigNumberTween = tweened(0, {
-    duration: 1000,
-    delay: 200,
-    easing: cubicIn,
-  });
-  $: bigNumberTween.set(bigNumber);
+
+  $: bigNumber =
```

Comment:
you can format the tweened value directly and it shoudl work, but truthfully I'm going to be removing this file so I will take in this logic and apply it in my branch.